#swtch 1329/enrollment dropdown

whitneypurdum opened the pull request:
- Replaces Enrollment dropdown with buttons
- https://energyweb.atlassian.net/jira/software/projects/SWTCH/boards/54?selectedIssue=SWTCH-1329

<img width="1405" alt="Screen Shot 2022-05-13 at 12 06 22 PM" src="https://user-images.githubusercontent.com/83423498/168323764-d3738f3b-aca3-4b1b-8aca-ac5e5f50af41.png">
<img width="1348" alt="Screen Shot 2022-05-13 at 12 06 34 PM" src="https://user-images.githubusercontent.com/83423498/168323775-79199d74-1555-4e60-aadb-75cea803b7c2.png">

Commit:
feat: update enrollment dropdown to use buttons

Commit:
fix: apply prettier formatting

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/energywebfoundation/switchboard-dapp?pullRequest=647) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/energywebfoundation/switchboard-dapp?pullRequest=647) before we can accept your contribution.<br/><hr/>**whitneypurdum** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/energywebfoundation/switchboard-dapp?pullRequest=647) it.</sub>

whitneypurdum commented on src/app/shared/components/table/enrolment-list-filter/enrolment-list-filter.component.scss:
```diff
@@ -0,0 +1,7 @@
+.filter-button {
+  margin-right: 10px;
+  background-color: gray;
```
I didn't have rgbs or anything specific for this, but when I do I guess I should add this to scss variables @dawidgil

dawidgil replied:
They are in theme files. I refactored it to use css variables as it make more sense (for me) to have it this way, because we are supporting 2 different styles for 2 applications (Switchboard and Stedin)

whitneypurdum replied:
@dawidgil yeah, I just am not sure if the color I'm supposed to use is in there, because there's no RGBs on the mocks atm. I will just use the --button-disabled from theme and can change later if it doesn't look right!

whitneypurdum committed:
fix: remove log

whitneypurdum committed:
remove log

whitneypurdum committed:
:wq

iMerge branch '#SWTCH-1329/EnrollmentDropdown' of https://github.com/energywebfoundation/switchboard-dapp into #SWTCH-1329/EnrollmentDropdown

whitneypurdum committed:
fix: remove log

whitneypurdum committed:
Merge branch '#SWTCH-1329/EnrollmentDropdown' of https://github.com/energywebfoundation/switchboard-dapp into #SWTCH-1329/EnrollmentDropdown

whitneypurdum commented:
> @whitneypurdum 

could you adjust your github config and commits (or maybe sign the CLA?) to remove the CLA bot warning please?

<img width="415" alt="Screen Shot 2022-05-13 at 2 44 22 PM" src="https://user-images.githubusercontent.com/83423498/168359965-8c111c82-0df7-4b9f-8d2b-530ea9c0f6dd.png">

whitneypurdum committed:
fix: use color var from default theme
